pom.xml

```diff
@@ -266,23 +201,21 @@
       <version>1.16</version>
     </dependency>
 
-
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-      <version>4.12</version>
```

Comment:
Version in `dependencyManagement` of parent POM. Left here for `Matchers` classpath issues.